vehicles/descriptions/sam_description/urdf/sam_auv.urdf.xacro

```diff
@@ -6,11 +6,11 @@
   <xacro:property name="prop_nozzle" value="file://$(find sam_description)/mesh/SAM_NOZZLE.dae"/>
   <xacro:property name="front_prop" value="file://$(find sam_description)/mesh/SAM_FRONTPROP.dae"/>
   <xacro:property name="back_prop" value="file://$(find sam_description)/mesh/SAM_BACKPROP.dae"/>
-
+  <xacro:property name="ns" value="sam"/>
```

Comment:
i see that all this xacro is doing that a URDF does not, is to put a prefix (ns) in front of everything. 
the robot state publisher can do this with a parameter. see: https://github.com/ros/robot_state_publisher/tree/ros2?tab=readme-ov-file#parameters

so maybe an xacro is not needed at all?